src/main/java/seedu/address/model/common/Date.java

```diff
@@ -1,32 +1,45 @@
 package seedu.address.model.common;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
 
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
 
 public class Date {
+    public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd/mm/yyyy");
```

Comment:
DateTimeFormatter and Regex validation format should follow that of the UG: YYYY-MM-DD